src/test/java/wooteco/subway/path/domain/fare/creationstrategy/DistanceBasedCreationStrategyTest.java

```diff
@@ -0,0 +1,41 @@
+package wooteco.subway.path.domain.fare.creationstrategy;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import wooteco.subway.path.domain.fare.Fare;
+
+class DistanceBasedCreationStrategyTest {
+
+    @DisplayName("거리별 기본 요금 계산")
+    @MethodSource("distanceFareValues")
+    @ParameterizedTest
+    void defaultRate(int distance, Fare expectedFare) {
+        // given
+        DistanceBasedCreationStrategy distanceBasedCreationStrategy = new DistanceBasedCreationStrategy();
+
```

Comment:
```suggestion
        FareCreationStrategy distanceBasedCreationStrategy = new DistanceBasedCreationStrategy();

```

구체보다 인터페이스를 사용하는 습관이 좋습니다 :)